src/main/java/com/team4/testingsystem/repositories/TestsRepository.java

```diff
@@ -52,8 +52,8 @@
 
     @Transactional
     @Modifying
-    @Query(value = "UPDATE Test t SET t.verifiedAt = ?1 where t.id = ?2")
-    int updateEvaluation(Instant updateDate, Long id);
+    @Query(value = "UPDATE Test t SET t.verifiedAt = ?1, t.status = 'VERIFIED' where t.id = ?2")
+    int update(Instant updateDate, Long id);
```

Comment:
let's rename it to something like 'verify'/`coachSubmit` or smth. like that. 'update' seems too unclear